src/bindings/python/make_binding.py

```diff
@@ -209,9 +209,12 @@ def process_header(f, including_path="."):
 {includes}
 
 ffi.cdef("""
+typedef int... ptrdiff_t;
```

Comment:
Can this `make_binding.py` file be removed?  I'm not seeing any references to it anywhere anymore.

Reply:
It can, I had been leaving it because I couldn't remember if there were any dependencies on it from outside of core.  I'll pull it out while I do my rebase this morning.